apps/modernization-ui/src/components/FormInputs/DatePickerInput.tsx

```diff
@@ -113,7 +110,9 @@ const InternalDatePicker = ({
             onBlur={onBlur}
             onKeyDown={handleKeyDown}
             onChange={handleOnChange(onChange)}
-            className={classNames(styles.input, className)}
+            className={classNames(className)}
+            validationStatus={errorMessage ? 'error' : undefined}
+            aria-describedby={errorMessage ? `${errorMessage}-message` : undefined}
```

Comment:
question:  What are these properties used for?

Reply:
I used validationStatus to indicate the state of the form input on error and add styles then when there is an error present when its set to error its set to undefined and I applied the styles for error then to update the fields border to red.
I got it from here `https://trussworks.github.io/react-uswds/?path=/docs/components-date-input-docs`